src/csharp/SDK/Device.cs

```diff
@@ -167,6 +167,32 @@ public HardwareVersion Version
             }
         }
 
+        /// <summary>
+        /// Gets the native handle.
+        /// </summary>
+        /// <remarks>This is the value of the k4a_device_t handle of the native library.
+        ///
+        /// This handle value can be used to interoperate with other native libraries that use
+        /// Azure Kinect objects.
+        ///
+        /// When using this handle value, the caller is responsible for ensuring that the
+        /// Device object does not become disposed.</remarks>
```

Comment:
Would you want to add that locking on this object will gaurentee that it isn't disposed while you are using the handle?